.github/actions/use-ray-cluster/action.yml

```diff
@@ -0,0 +1,23 @@
+name: "Set up local Ray cluster"
+description: "Set up local Ray cluster"
+inputs:
+  ray-memory:
+    description: "Ray store memory size"
+    required: true
```

Comment:
```suggestion
    required: false
    default: 1000000000
```

Reply:
I wanted to keep the number for memory limit in one place, namely in `ci.yml`

Reply:
ok

Reply:
I have reverted the change, it isn't playing nicely with other tests for some reason, and isn't saving much of a copy-paste anyway 😞